src/rules/relative-font-units.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * @fileoverview Enforce the use of relative units for font size.
+ * @author Tanuj Kanti
+ */
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/**
+ * @import { CSSRuleDefinition } from "../types.js"
+ * @typedef {"allowedFontUnits"} RelativeFontUnitsMessageIds
+ * @typedef {[{allow?: string[]}]} RelativeFontUnitsOptions
+ * @typedef {CSSRuleDefinition<{ RuleOptions: RelativeFontUnitsOptions, MessageIds: RelativeFontUnitsMessageIds}>} RelativeFontUnitsRuleDefinition
+ */
+
+//-----------------------------------------------------------------------------
+// Helpers
+//-----------------------------------------------------------------------------
+
+const relativeFontUnits = [
+	"%",
+	"cap",
+	"ch",
+	"em",
+	"ex",
+	"ic",
+	"lh",
+	"rcap",
+	"rch",
+	"rem",
+	"rex",
+	"ric",
+	"rlh",
+];
+
+const fontSizeIdentifiers = new Set([
+	"xx-small",
+	"x-small",
+	"small",
+	"medium",
+	"large",
+	"x-large",
+	"xx-large",
+	"smaller",
+	"larger",
+]);
+
+//-----------------------------------------------------------------------------
+// Rule Definition
+//-----------------------------------------------------------------------------
+
+/** @type {RelativeFontUnitsRuleDefinition} */
+export default {
+	meta: {
+		type: "suggestion",
+
+		docs: {
+			description: "Enforce the use of relative font units",
+			recommended: false,
+			url: "https://github.com/eslint/css/blob/main/docs/rules/relative-font-units.md",
+		},
+
+		schema: [
+			{
+				type: "object",
+				properties: {
+					allow: {
```

Comment:
Should we call it `allowUnits`?

```suggestion
					allowUnits: {
```

Reply:
Make sense to me! would like to confirm from @nzakas.

Reply:
I think "units" can be implied from the name of the rule. As there is nothing else the rule deals with besides units, I think `allow` is fine.

Reply:
Done!